laravel/resources/views/partials/warning/concurrent-slot.blade.php

```diff
@@ -0,0 +1,11 @@
+<span>
+@if(isset($admin) && $admin === true)
+    {{ $user_name }} is currently signed up for another
+    <a href=" {{ env('SITE_URL') }} /slot/{{ $concurrent_slot_id }}/view">overlapping shift</a>.
```

Comment:
this space in the href url is causing problems, eliminate the space so we can actually see the page. infact all of the hrefs in this portion of the code

Reply:
@Meleeman01 Can you fix this issue?